src/patternfly/components/Expandable/expandable-toggle.hbs

```diff
@@ -0,0 +1,12 @@
+<button class="pf-c-expandable__toggle{{#if expandable-toggle--modifier}} {{expandable-toggle--modifier}}{{/if}}"
```

Comment:
I wonder if the toggle area for this button should match our minimum target for buttons, dropdowns, etc? If so, I wonder if it would be acceptable to use a `.pf-c-button.pf-m-link` here?